cmd/kube-proxy/app/server.go

```diff
@@ -627,7 +627,7 @@ func serveMetrics(bindAddress, proxyMode string, enableProfiling bool, errCh cha
 		fmt.Fprintf(w, "%s", proxyMode)
 	})
 
-	//lint:ignore SA1019 See the Metrics Stability Migration KEP
+	//nolint:staticcheck // SA1019 See the Metrics Stability Migration KEP
```

Comment:
Why is this turning off a check that is already turn off in `.golangci.yaml`?

Reply:
because I did a search and replace in all files replacing the all comments by the new ones.
We should get this merged to have the golangci-linter working, and then we can start cleaning all these things in parallel, it is also a good opportunity for new contributors ...